views/partials/barista_shifts_table.ejs

```diff
@@ -0,0 +1,73 @@
+<table class="table table-compact w-full">
+  <thead>
+    <tr>
+      <th scope="col">Date</th>
+      <th scope="col">Location</th>
+      <th scope="col">time / duration</th>
+      <th scope="col">Wage</th>
+      <th scope="col">action</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% if (shifts.length == 0 ) { %>
+      <tr>
+        <th colspan="5" class="text-center">There is no shift to display</th>
```

Comment:
I think you used to have this be a different string based on which version of the table it is. You don't have to sacrifice that for the partial. You could pass a string like `noDataLabel` as a "prop" to this partial and display that string here!